modular_skyrat/modules/records_on_examine/code/examine.dm

```diff
@@ -0,0 +1,15 @@
+/mob/living/carbon/human/get_medhud_examine_info(mob/living/user, datum/record/crew/target_record)
+	. = ..()
+
+	//BUBBER EDIT ADDITION BEGIN - EXAMINE RECORDS
+	. += "<a href='?src=[REF(src)];hud=m;medrecords=1;examine_time=[world.time]'>\[View medical records\]</a>"
+	. += "<a href='?src=[REF(src)];hud=m;genrecords=1;examine_time=[world.time]'>\[View general records\]</a>"
+	//BUBBER EDIT END
+
+/mob/living/carbon/human/get_sechud_examine_info(mob/living/user, datum/record/crew/target_record)
+	. = ..()
+
+	//BUBBER EDIT ADDITION BEGIN - EXAMINE RECORDS
+	. += "<a href='?src=[REF(src)];hud=s;secrecords=1;examine_time=[world.time]'>\[View past security records\]</a>"
+	. += "<a href='?src=[REF(src)];hud=s;genrecords=1;examine_time=[world.time]'>\[View general records\]</a>"
+	//BUBBER EDIT END
```

Comment:
Actually, one more thing, is there a better way to do it without duping the `View general records` on both the procs?

Reply:
probably, but i dont know how and i have a sinking feeling it might be a bit messy